kb-api-ws/src/test/java/dk.kb.api/utilities/RestUtilTest.java

```diff
@@ -1,4 +1,43 @@
 package dk.kb.api.utilities;
 
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
```

Comment:
Please add a note explaining why this is not a standard Unit-test (I guess it it because it requires a running Solr server).

Reply:
Det er rigtig gættet og det vil jeg gøre. Men jeg er endelig ikke helt færdig med test